compose-files/provider.yml

```diff
@@ -24,5 +24,6 @@ services:
       IPFS_GATEWAY: ${PROVIDER_IPFS_GATEWAY}
       OCEAN_PROVIDER_TIMEOUT: '9000'
       OPERATOR_SERVICE_URL: ${OPERATOR_SERVICE_URL}
+      AQUARIUS_URL: ${AQUARIUS_URL:-http://172.15.0.5:5000}
```

Comment:
we have `AQUARIUS_URL` & `AQUARIUS_URI`? https://github.com/oceanprotocol/barge/blob/v3/start_ocean.sh#L84

We should only have one, so just use `AQUARIUS_URI` everywhere, no need to set another default here